src/borg/_hashindex.c

```diff
@@ -422,8 +434,10 @@ hashindex_set(HashIndex *index, const void *key, const void *value)
     if(idx < 0)
     {
         /* we don't have the key in the index we need to find an appropriate address */
+      /* inserts ++;  //TRACE_PERF */
```

Comment:
indentation issue here? tab / space mix? maybe check that globally and also your editor settings.